src/dashboard/Data/Browser/BrowserTable.react.js

```diff
@@ -115,6 +115,64 @@ export default class BrowserTable extends React.Component {
         (rowWidth, { visible, width }) => visible ? rowWidth + width : rowWidth,
         this.props.onAddRow ? 210 : 0
       );
+      let editCloneRows = null;
```

Comment:
Why `null`?

Reply:
Initially `null`, as we don't want to render any rows.

Reply:
Wouldn't it be enough to declare it and check for `undefined`?

```
let editCloneRows;

if (editCloneRows) ...

```

Reply:
Yes, we can declare it & check for `undefined`. but, I think this variable is going to hold html node element. So, is check necessary here? we can also directly put something like this
``` 
let edilCloneRows;
...
<div>
 {editCloneRows}
</div>
```

Reply:
Sure, I just wanted to know whether the `= null` is necessary. If not then I suggest to remove it to keep the code simple. There is also another `= null` below.

Reply:
>  There is also another `= null` below.

Yes, I followed this exact reference. :nerd_face:
I don't think that `=null` is necessary, should I remove it then ?

Reply:
If it's unnecessary, please remove it. Otherwise one may asume there was a reason why this is set to `null`, which is a distinct value that is different from `undefined`.

Reply:
Sure, please see https://github.com/parse-community/parse-dashboard/pull/1697/commits/be85c8e6349540ae458c77792b749a7280b98436

Reply:
The `username` field still is blank instead of `(undefined)`:
![image](https://user-images.githubusercontent.com/5673677/126083251-051ff034-b553-49f9-923e-a0045ef3cf80.png)

Will that resolve with merge of #1720 as you [commented](https://github.com/parse-community/parse-dashboard/pull/1697#issuecomment-870164371), or is this a separate issue?